jobs/JGFS_ATMOS_FSU_GENESIS

```diff
@@ -33,25 +33,18 @@ export PYTHONPATH=${USHens_tracker}/FSUgenesisPY:${PYTHONPATH}
 ##############################################
 # Define COM and Data directories
 ##############################################
-export COMIN=${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}
-export COMINgfs=${COMIN}
-#export gfsdir=${COMINgfs}
-export gfsdir=${ROTDIR}
-export COMINgdas=${COMIN}
-export gdasdir=${COMINgdas}
-export COMOUT=${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}
-export COMINsyn=${COMINsyn:-$(compath.py ${envir}/com/gfs/${gfs_ver})/syndat}
-
-if [ ${RUN_ENVIR} = "nco" ]; then
-    export COMOUThur=${COMROOTp1}/hur/${envir}/global
-    export COMOUTatcf=${COMROOTp1}/nhc/${envir}/atcf
-    mkdir -m 775 -p ${COMOUThur} ${COMOUTatcf}
-else
-#    export COMOUThur=$COMOUT
-#    export COMOUTatcf=$COMOUT
-    export COMOUThur=${DATA}
-    export COMOUTatcf=${DATA}
-fi
+YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_GENESIS
+YMD=${PDY} HH=${cyc} GRID="0p25" generate_com -rx COM_ATMOS_GRIB_0p25:COM_ATMOS_GRIB_GRID_TMPL
+
+# The following variables are used by the tracker scripts which are outside
+#   of global-workflow and therefore can't be standardized at this time
+export COMIN=${COM_ATMOS_GRIB_0p25}
+export gfsdir=${COM_ATMOS_GRIB_0p25}
+export COMINgfs=${COM_ATMOS_GRIB_0p25}
+
```

Comment:
- Does the genesis script use all of `COMIN`, `gfsdir`, and `COMINgfs`?
- `gdasdir` (and `COMINgdas`) are no longer being defined.

Reply:
> * Does the genesis script use all of `COMIN`, `gfsdir`, and `COMINgfs`?

Yes, there is a mixture of all three across both the `TC_tracker` (dev) and `ens_tracker` (ops) packages. Note, the FSU_GENESIS job is not tested so further changes to the JJOB script are potentially needed.

> * `gdasdir` (and `COMINgdas`) are no longer being defined.

The FSU genesis job is only run for `gfs` in operations so we don't need the `gdas` versions of those path variables.